docs/develop/update/postgres.rst

```diff
@@ -48,14 +50,118 @@ Add your configuration
 
 The configuration file will be in the ``/etc/postgresql/11/main/conf.d/`` directory on the server (for PostgreSQL version 11).
 
+.. _pg-setup-backups:
+
+Set up backups
+--------------
+
+We use pgbackrest to create and manage backups stored in the cloud.
+Salt will install and configure pgbackrest if ``postgres:backup`` is defined in pillar.
```

Comment:
Use correct capitalization of pgBackRest throughout, unless referring to the command.

```suggestion
Salt will install and configure pgBackRest if ``postgres:backup`` is defined in Pillar data.
```